### ISSUE TITLE
Missing revocation endpoint auth method

### ISSUE THREAD

**Issue by ivysdad** [dnplus/streamlit-oauth#51]
Everything was working a couple days ago in production when the app was deployed to streamlit cloud, but now I'm getting this error: 

"MissingRevokeTokenAuthMethodError: Missing revocation endpoint auth method."

From the outputs: 
```
/home/adminuser/venv/lib/python3.11/site-packages/httpx_oauth/oauth2.py:197   

  in __init__                                                                   

                                                                                

    194 │   │   │   revoke_token_endpoint is not None                           

    195 │   │   │   and revocation_endpoint_auth_method is None                 

    196 │   │   ):                                                              

  ❱ 197 │   │   │   raise MissingRevokeTokenAuthMethodError()
```

**Comment by dnplus** [comment]
refer to #49 

Fixed httpx-oauth version in 0.1.13

**Comment by ivysdad** [comment]
> refer to #49
> 
> Fixed httpx-oauth version in 0.1.13

this worked, thank you so much!!!!!!! closing the ticket. <3
